test/rulesets/Base/arraymath.jl

```diff
@@ -132,9 +132,19 @@
     end
 
     @testset "/ and \\ Scalar-AbstractArray" begin
-        A = randn(3, 4, 5)
+        A = round.(10 .* randn(3, 4, 5), digits=1)
         test_rrule(/, A, 7.2)
         test_rrule(\, 7.2, A)
+
+        # test_rrule(/, A, 7.2+8.3im)
```

Comment:
Do we want the commented out tests?

Reply:
They fail because, when `b` is complex, the gradient w.r.t. real `A` also becomes complex. We should fix this, but I'm not sure that adding logic to individual rules to check this is worthwhile, probably better to add an overall projection step.

Reply:
Aha, thanks, makes sense. Should we mark them `@test_broken` and link an issue/this conversation instead?
I will work on adding a projection in the next weeks

Reply:
Maybe simplest to delete. Depending on exactly how projection gets done, I guess it's not obvious whether it would fix these tests. And, the same concern applies to every rule which accepts multiple arguments.